### PR TITLE
Fix doc push

### PR DESCRIPTION
**Pull request by m-vdb**
**Proposed changes**:
- Fix "Publish Documentation" workflow
- I've also re-run the fixed workflow locally to make sure the docs are updated. Build is running [here](https://app.netlify.com/teams/rasahq/builds/610ab3c915a9c90007571183)

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/main/changelog) for instructions)
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

cc @aeshky as you ran the release of 2.8.2 and uncovered the issue 😄 thanks!

### CODE REVIEW

File: scripts/push_docs_to_branch.sh [hunk 1]
```diff
@@ -48,6 +48,9 @@ then
     EXISTING_VERSION=$NEW_VERSION
 fi
 
+# install yarn dependency in the temp directory
+cd $TMP_DOCS_FOLDER/docs && yarn install && cd - || exit 1
```
Comment (m-vdb) [inline scripts/push_docs_to_branch.sh:52]: no idea why it didn't fail before, maybe some sort of global cache in `yarn`?! not an expert

**Comment by m-vdb** [comment]
I've already requested someone from infra -- let's keep Tobias out of this one 😄

File: scripts/push_docs_to_branch.sh (hunk 1 above)
Comment (mprazz) [reply scripts/push_docs_to_branch.sh:52]: I have no clue either tbh :sweat_smile: I'm way less aware of the `rasa` side compared to the `rasa-x` side :grimacing:

**Review by mprazz — APPROVED** [review]
LGTM :medal_military:

**Comment by mprazz** [comment]
if it works now, that's the important part -- doing a yarn install is not a destructive action so I don't think we should be overly concerned about this.

File: scripts/push_docs_to_branch.sh (hunk 1 above)
Comment (m-vdb) [reply scripts/push_docs_to_branch.sh:52]: I do think we'll have the same pb for Rasa X 😄

**Pull request merged**